GooglePlayInstant/Editor/PlayInstantBuildConfiguration.cs

```diff
@@ -125,15 +121,27 @@ public static void SetInstantBuildType()
         public static void SetInstalledBuildType()
         {
             var scriptingDefineSymbols = GetScriptingDefineSymbols();
-            if (IsPlayInstantScriptingSymbolDefined(scriptingDefineSymbols))
+            if (IsScriptingSymbolDefined(scriptingDefineSymbols, PlayInstantScriptingDefineSymbol))
             {
                 SetScriptingDefineSymbols(scriptingDefineSymbols.Where(sym => sym != PlayInstantScriptingDefineSymbol));
             }
         }
 
-        private static bool IsPlayInstantScriptingSymbolDefined(string[] scriptingDefineSymbols)
+        /// <summary>
+        /// Adds the specified scripting define symbol for Android, but only if it isn't already defined.
+        /// </summary>
+        public static void AddScriptingDefineSymbol(string symbol)
+        {
+            var scriptingDefineSymbols = GetScriptingDefineSymbols();
+            if (!IsScriptingSymbolDefined(scriptingDefineSymbols, symbol))
+            {
+                SetScriptingDefineSymbols(scriptingDefineSymbols.Concat(new[] {symbol}));
+            }
+        }
+
+        private static bool IsScriptingSymbolDefined(string[] scriptingDefineSymbols, string symbol)
```

Comment:
Maybe reduce this signature to `private static bool IsScriptingSymbolDefined(string symbol)` and call `GetScriptingDefineSymbol()` inside the function.